src/gsi/obsmod.F90

```diff
@@ -621,8 +622,8 @@ module obsmod
   integer(i_kind) ntilt_radarfiles,tcp_posmatch,tcp_box
 
   logical ::  ta2tb
-  logical ::  doradaroneob
-  logical :: vr_dealisingopt, if_vterminal, if_model_dbz, inflate_obserr, if_vrobs_raw, if_use_w_vr, l2rwthin
+  logical ::  doradaroneob,dofedoneob
+  logical :: vr_dealisingopt, if_vterminal, if_model_dbz,if_model_fed, innov_use_model_fed,inflate_obserr, if_vrobs_raw, if_use_w_vr, l2rwthin
```

Comment:
@hongli-wang Can you add a description for innov_use_model_fed somewhere in the code? It is not easy to understand what this variable is used for. Thank you.